pendulum_control/include/pendulum_control/rtt_executor.hpp

```diff
@@ -15,6 +15,8 @@
 #ifndef PENDULUM_CONTROL__RTT_EXECUTOR_HPP_
 #define PENDULUM_CONTROL__RTT_EXECUTOR_HPP_
 
+#include <string.h>
```

Comment:
ok, I've changed this. I pushed the same fix to rclcpp last night, so I'll change it in that pull request too. Warning @wjwwood that I'm pushing to his branch.